index.bs

```diff
@@ -36,6 +36,7 @@ Former Editor: Rolf Lindemann, w3cid 84447, Nok Nok Labs, rolf@noknok.com
 !Contributors: <a href="mailto:agl@google.com">Adam Langley</a> (Google)
 !Contributors: <a href="mailto:mandyam@qti.qualcomm.com">Giridhar Mandyam</a> (Qualcomm)
 !Contributors: <a href="mailto:mattmil3@cisco.com">Matthew Miller</a> (Cisco)
+!Contributors: <a href="mailto:pascoe@apple.com">Pascoe</a> (Apple)
```

Comment:
No first name? (I don't know whether W3C has anything against omitting it, I just noticed it stuck out a bit)

Reply:
That is my full government name.

Reply:
Oh, my apologies!